include/ck/tensor_operation/gpu/grid/gridwise_gemm_bias_add_reduce_xdl_cshuffle_v1.hpp

```diff
@@ -119,22 +119,22 @@ template <typename FloatAB,
           typename FloatC0,
           typename FloatC1,
           typename FloatReduceAcc,
-          typename DPtrsGlobal,
+          typename ReducePtrsGlobal,
```

Comment:
naming: "D" is more concise, and would be clearer than "Reduce" if you define what this Device op does by writing down the formula in comment

Also, this gridwise gemm should be updated to be similar to
"gridiwse gemm multiple D" and use "multiple Ds" instead of "Bias and Add"